recipe/upload_or_check_non_existence.py

```diff
@@ -16,19 +16,24 @@
 from conda_build.config import Config
 
 
+def bldpkg_path_wrapper(meta):
+    try:
+        config = Config()
+        fname = bldpkg_path(meta, config)
+    except TypeError:
+        # conda-build < 2.0.0 takes a single config argument
```

Comment:
There is more to come. Maybe we should take a second look at latest conda-build.

See https://github.com/SciTools/conda-build-all/pull/68#issuecomment-267612652